### PR TITLE
Twitter Icon Updated to X using react-icons (not using any svg)

### LINKED ISSUE #772: Request to Update Twitter Icon to X
### Describe the bug

I would like to propose a change to the current Twitter icon. As icons play a significant role in representing the project and creating a visual identity, I believe updating the Twitter icon could enhance the overall aesthetics and recognition of the website.

WeMakeDevs - Google Chrome 8_30_2023 11_05_41 PM

### To Reproduce

_No response_

### Expected Behavior

- Change the old logo with the new one.

### Screenshot/ Video

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by maverick-farhan**
## Fixes Issue

This PR fixes the following issues:

closes #772

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.

## Screenshots
twiiter-icon-changed-to-X  
screenshot

code-screenshot

Add all the screenshots which support your changes  

Refactor: Updated the Twitter icon in the Footer component. The previously used FaTwitter icon from react-icons/fa has been replaced with FaXTwitter icon from react-social-icon package.

### Summary by CodeRabbit

- Style: Updated the Twitter icon in the footer to use `FaXTwitter` instead of `FaTwitter`.
- Style: Changed the hover color for the Twitter link in the footer to `#FAF3F0` from `#1DA1F2` for better visual consistency.

### CODE REVIEW

**Review by maverick-farhan — COMMENTED** [review]
Updated twitter icon to X would you please review my changes. <3

File: src/components/layout/Footer.jsx [hunk 1]
```diff
@@ -180,9 +180,9 @@ const links = [
   {
     name: 'twitter',
     link: 'https://twitter.com/WeMakeDevs',
-    icon: <FaTwitter />,
+    icon: <FaXTwitter />,
     ariaLabel: 'Visit WeMakeDevs on Twitter',
-    className: 'hover:text-[#1DA1F2]',
+    className: 'hover:text-[#FAF3F0]',
```
Comment (AnuragThePathak) [inline src/components/layout/Footer.jsx:185]: Need for this line?

Comment (maverick-farhan) [reply src/components/layout/Footer.jsx:185]: Yeah because old Twitter color is blue
And new X icon is not

Comment (maverick-farhan) [reply src/components/layout/Footer.jsx:185]: Bit I can change as you see fit

Comment (maverick-farhan) [reply src/components/layout/Footer.jsx:185]: This is for color, as we can choose any suitable color in future

**Review by AnuragThePathak — APPROVED** [review]

**Pull request merged**